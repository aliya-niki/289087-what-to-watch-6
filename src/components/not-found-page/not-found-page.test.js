import React from 'react';
import {render, screen} from '@testing-library/react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';
import NotFoundPage from './not-found-page';

it(`NotFoundPage should render correctly`, () => {
  const history = createMemoryHistory();

  const {container} = render(
      <Router history={history}>
        <NotFoundPage />
      </Router>
  );

  expect(screen.getByText(`404. Page not found`)).toBeInTheDocument();
  expect(screen.getByText(`Вернуться на главную`)).toBeInTheDocument();

  expect(container).toMatchSnapshot();
});
