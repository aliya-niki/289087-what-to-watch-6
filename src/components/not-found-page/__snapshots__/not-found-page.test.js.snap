// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage should render correctly 1`] = `
<div>
  <h1>
    404. Page not found
  </h1>
  <a
    href="/"
  >
    Вернуться на главную
  </a>
</div>
`;
